pulsar-client/src/main/java/org/apache/pulsar/client/api/Message.java

```diff
@@ -20,6 +20,9 @@
 package org.apache.pulsar.client.api;
 
 import java.util.Map;
+import java.util.Optional;
+
+import org.apache.pulsar.client.impl.EncryptionContext;
```

Comment:
@rdhabalia The `EncryptionContext` should be moved to `api` package as weel

Reply:
yes, I also realized it.. making this change.